### PR TITLE
Lit v0.8.0 alpha release

### PR DESCRIPTION
**Pull request by ellemouton**
prepping for next release so long

- [x] lnd 0.15.1-beta
- [x] lndclient 0.15.1
- [x] loop v0.20.1-beta 
- [x] pool v0.5.8-alpha

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,19 +13,19 @@ require (
 	github.com/lightninglabs/aperture v0.1.18-beta
 	github.com/lightninglabs/faraday v0.2.8-alpha.0.20220624141723-ddd3cd123e62
 	github.com/lightninglabs/lightning-node-connect v0.1.12-alpha
-	github.com/lightninglabs/lndclient v0.15.0-10
-	github.com/lightninglabs/loop v0.19.1-beta.0.20220623090540-08209f61e304
-	github.com/lightninglabs/loop/swapserverrpc v1.0.1
+	github.com/lightninglabs/lndclient v0.15.1-1
```
Comment (guggero) [inline go.mod:16]: Should use `v0.15.1-5` which allows you to remove the `.rc1` suffix from the `lnd` version (without go mod changing it back).

File: go.mod [hunk 2]
```diff
@@ -13,19 +13,19 @@ require (
...
+	github.com/lightninglabs/loop v0.20.1-beta
+	github.com/lightninglabs/loop/swapserverrpc v1.0.3
```
Comment (guggero) [inline go.mod:18]: I just pushed a tag for Pool: `v0.5.8-alpha`.

File: README.md [hunk 3]
```diff
@@ -119,6 +126,7 @@ The following table shows the supported combinations:
 
 | LiT              | LND          | Loop         | Faraday      | Pool         |
 |------------------|--------------|--------------|--------------|--------------|
+| **v0.8.0-alpha** | v0.15.1-beta | v0.20.1-beta | v0.2.8-alpha | v0.5.7-alpha |
```
Comment (guggero) [inline README.md:129]: Don't forget to bump the Pool version after updating it in the go.mod.

**Review by guggero — COMMENTED** [review]
Looks good, will do a final round once we bumped to Pool `v0.5.8-alpha`.

**Review by guggero — APPROVED** [review]
utACK, LGTM :tada:

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 👌

Tested Loop & Pool with no issues. Nice work

**Pull request merged**